### ISSUE TITLE
Content Processor - SoundEffectProcessor failure

### ISSUE THREAD

**Issue by monobeginner** [MonoGame/MonoGame#5894]
Hey,

I'm facing issue with processing of sound effect (only one wav file from many). Thrown exception:
```
/buttonClick.wav: error: Processor 'SoundEffectProcessor' had unexpected failure!
System.DivideByZeroException: NastĄpia pr˘ba podzielenia przez zero.
   w Microsoft.Xna.Framework.Content.Pipeline.Audio.AudioContent.ConvertFormat(ConversionFormat formatType, ConversionQuality quality, String saveToFile)
   w Microsoft.Xna.Framework.Content.Pipeline.Processors.SoundEffectProcessor.Process(AudioContent input, ContentProcessorContext context)
   w Microsoft.Xna.Framework.Content.Pipeline.ContentProcessor`2.Microsoft.Xna.Framework.Content.Pipeline.IContentProcessor.Process(Object input, ContentProcessorContext context)
   w MonoGame.Framework.Content.Pipeline.Builder.PipelineManager.ProcessContent(PipelineBuildEvent pipelineEvent)
```

Verified with MonoGame Pipeline tool 3.5.0.826

In attachment you can find this wav file (zipped archive). I would like to mention that it was working some time ago (on the same MG Pipeline version), before I reinstalled system (Windows 8.1). Any workaround for this issue ? I would like to keep this version of MonoGame because I have bigger issues with 3.7 version (please follow http://community.monogame.net/t/bug-paths-format-is-not-supported/9589)

[buttonClick.zip](https://github.com/MonoGame/MonoGame/files/1246436/buttonClick.zip)

**Comment by tomspilman** [comment]
@KonajuGames ?

**Comment by Jjagg** [comment]
I just checked and the .wav builds succesfully in latest develop due to @KonajuGames fixes.

@monobeginner It would be better to fix your other issue in the develop branch. Could you open an issue for it? You could try to apply the patches that fixed the wav issue in your 3.5 version, but that might come with some headscratching.
